src/main/java/uk/gov/hmcts/reform/roleassignment/controller/endpoints/CreateAssignmentController.java

```diff
@@ -73,11 +69,8 @@ public CreateAssignmentController(CreateRoleAssignmentOrchestrator createRoleAss
     public ResponseEntity<Object> createRoleAssignment(@RequestHeader(value = "x-correlation-id", required = false)
                                                                String correlationId,
         @Validated
-        @RequestBody(required = true) AssignmentRequest assignmentRequest) throws ParseException {
+        @RequestBody AssignmentRequest assignmentRequest) throws ParseException {
```

Comment:
the required field is removed so default would be set to false.

Reply:
Hi Aashish, the default is true only
public @interface RequestBody {

	/**
	 * Whether body content is required.
	 * <p>Default is {@code true}, leading to an exception thrown in case
	 * there is no body content.

Reply:
ok, I thought default is usually false.